packages/manager/modules/pci/src/projects/project/notebooks/add/add.html

```diff
@@ -1,13 +1,136 @@
+
+    <!--Notebook Configuration-->
+    <oui-step-form
+        name="notebook_settings"
+        data-header="{{:: 'pci_notebook_add_configuration_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookConfiguration = true"
+        data-on-submit="$ctrl.displayNotebookConfiguration = false"
+        data-prevent-next="true"
+    >
+        <notebook-configuration
+            data-display="$ctrl.displayNotebookConfiguration"
+            data-notebook="$ctrl.notebook"
+            data-editors="$ctrl.editors"></notebook-configuration>
+    </oui-step-form>
+
+    <!--Notebook Framework-->
+    <oui-step-form
+        data-name="notebook_framework"
+        data-header="{{:: 'pci_notebook_add_framework_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookFramework = true"
+        data-on-submit="$ctrl.displayNotebookFramework = false"
+        data-navigation="$ctrl.notebook.selected.framework.model"
+    >
+        <notebook-frameworks-list
+            data-display="$ctrl.displayNotebookFramework"
+            data-notebook="$ctrl.notebook"
+            data-frameworks="$ctrl.frameworks"></notebook-frameworks-list>
+    </oui-step-form>
+
+    <!--Notebook Privacy Settings-->
+    <oui-step-form
+        name="notebook_private_settings"
+        data-header="{{:: 'pci_notebook_add_privacy_settings_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookPrivacySettings = true"
+        data-on-submit="$ctrl.displayNotebookPrivacySettings = false"
+        data-navigation="$ctrl.notebook.selected.privacy"
+    >
+        <notebook-privacy-settings
+            data-display="$ctrl.displayNotebookPrivacySettings"
+            data-notebook="$ctrl.notebook"></notebook-privacy-settings>
+    </oui-step-form>
+
+    <!--Notebook Datacenter Location-->
+    <oui-step-form
+        name="notebook_datacenter_location"
+        data-header="{{:: 'pci_notebook_add_privacy_datacenter_location_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookDatacenterLocation = true"
+        data-on-submit="$ctrl.onLocationStepSubmit()"
+        data-navigation="$ctrl.notebook.selected.region"
+    >
+        <pci-project-regions-list
+            data-regions="$ctrl.regions"
+            data-selected-region="$ctrl.notebook.selected.region"
+            data-display-selected-region="$ctrl.displaySelectedRegion"
+        ></pci-project-regions-list>
+    </oui-step-form>
+
+    <!--Notebook Resources-->
+    <oui-step-form
+        name="notebook_resources"
+        data-header="{{:: 'pci_notebook_add_resources_title' | translate }}"
+        data-on-focus="$ctrl.displayNotebookResources = true"
+        data-on-submit="$ctrl.displayNotebookResources = false"
+        data-navigation="$ctrl.notebook.selected.resource.usage && $ctrl.notebook.selected.resource.flavor"
+    >
+        <notebook-resources
+            data-display="$ctrl.displayNotebookResources"
+            data-notebook="$ctrl.notebook"
```

Comment:
Same as above. Have a model attribute exposed, in order to store the selected resources

Reply:
Same as here:
https://github.com/ovh/manager/pull/5029#discussion_r655671141